common/lanelet2_extension/CMakeLists.txt

```diff
@@ -28,35 +28,41 @@ find_path(PUGIXML_INCLUDE_DIRS
   PATH_SUFFIXES pugixml
 )
 
-find_package(autoware_build_flags REQUIRED)
-
 find_package(PROJ4 REQUIRED)
 
+# Extract environment variable value
+set(LANELET2_EXTENSION_LOGGER_TYPE $ENV{LANELET2_EXTENSION_LOGGER_TYPE})
+
+if(LANELET2_EXTENSION_LOGGER_TYPE EQUAL 1 )
+
+  message("lanelet2_extension using std::cout and cerr for logging")
+  add_definitions(-DLANELET2_EXTENSION_LOGGER_TYPE=1)
+
+else() # Default to ROS logging
+
+  message("lanelet2_extension using roscpp for logging")
+  add_definitions(-DLANELET2_EXTENSION_LOGGER_TYPE=0) 
+  set(LOGGER_DEP roscpp)
+  
+endif()
+
 find_package(catkin REQUIRED COMPONENTS
-  amathutils_lib
-  roscpp
+  ${LOGGER_DEP}
```

Comment:
If this is undefined (the case where `LANELET2_EXTENSION_LOGGER_TYPE=1`) the CMAKE will just evaluate this to empty string right? It shouldn't error out attempting to process an undefined variable?

Reply:
Yeah I tested this case and it works fine by defaulting to 0